src/desktop/apps/artist/templates/meta.jade

```diff
@@ -39,3 +39,6 @@ if nav
     link( rel="prev", href="#{sd.APP_URL}#{rels.prev.href}" )
   if rels.next
     link( rel="next", href="#{sd.APP_URL}#{rels.next.href}" )
+
+//- Unica Font
+link( rel='stylesheet' type='text/css' href="#{sd.WEBFONT_URL + '/unica-webfonts.css'}")
```

Comment:
you can put this behind a `if ...` as well (for perf reasons), but thats #ohsominor